source/_integrations/gree.markdown

```diff
@@ -11,14 +11,51 @@ ha_codeowners:
 ha_domain: gree
 ---
 
-The Gree integration allows you to control a [Gree Smart HVAC](http://global.gree.com/).
+The Gree integration allows you to control a [Gree Smart HVAC](http://global.gree.com/) in Home Assistant.
+
+There is currently support for the following device types within Home Assistant:
+
+- [Climate](#climate)
+- [Switch](#switch)
  
 ## Configuration
 
 Go to the integrations page in your configuration and click on **new integration** -> **Gree**.
 
 Gree Smart compatible devices are detected and added automatically.
 
+<div class='note'>
+
+Devices must first be bound to WiFi using the Gree+ app, or app provided by the device manufacturer. It is not possible to connect devices to WiFi using this integration at the current time.
+
+</div>
+
 ## Supported models
 
-Any Gree Smart device working with the Gree+ app should be supported, including non-Gree branded devices, including some sold by Trane.
+Any Gree Smart device working with the Gree+ app should be supported, including non-Gree branded devices such some sold by:
+
+- Trane
+- Innova
+- Cooper & Hunter
+- Proklima
+
+## Climate
+
+The `gree` climate platform integrates Gree HVAC systems into Home Assistant, enabling control of setting the following parameters:
+
+- [**set_hvac_mode**](/integrations/climate/#service-climateset_hvac_mode)
+- [**target temperature**](/integrations/climate#service-climateset_temperature)
+- [**turn on/off**](/integrations/climate#service-climateturn_on)
+- [**fan mode**](/integrations/climate#service-climateset_fan_mode)
+- [**swing mode**](/integrations/climate#service-climateset_swing_mode)
+- [**set_preset_mode**](/integrations/climate#service-climateset_preset_mode)
```

Comment:
These are service names, referencing to code, and should be escaped using backticks, not emphasized as bold text.